core/modules/util/Error.h

```diff
@@ -40,6 +40,33 @@ namespace lsst {
 namespace qserv {
 namespace util {
 
```

Comment:
#ifndef in this file is non-conventional: UTIL_ERROR_H_, should be LSST_QSERV_UTIL_ERROR_H. While you are at it, could you also fix:

qana/CheckAggregation.h:#ifndef QANA_CHECKAGGREGATION_H_
qmeta/QMeta.h:#ifndef QMETA_QMETA_H
query/typedefs.h:#ifndef QUERY_TYPES_H_
util/Error.h:#ifndef UTIL_ERROR_H_